### PR TITLE
Update text component to use textlayoutmanager passed from ParagraphState

### PR DESCRIPTION
**Pull request by pprabhu-ng**
1. Use shared textlayoutmanager instead of having seperate textlayoutmanager for every text component.
Thus we use single instance of font collection manager, which would be one-time building.
This reduces the layout time.

2. Resolve the glog error 
Build RNS with "is_debug=false" to enable optimization.
disabling this flag, gives error in glog raw_logging.cc file with "unused-result" error. So configuring glog build with "Wno-unused-result" flag to avoid this error in glog thirdparty code.

### CODE REVIEW

File: ReactSkia/components/RSkComponentText.cpp [hunk 1]
```diff
@@ -46,11 +46,10 @@ void RSkComponentParagraph::OnPaint(SkCanvas *canvas) {
   /*    - use parent paragraph builder to add text & push style */
   /*    - draw the paragraph, when we reach the last fragment attachment*/
   if(parent) {
-      parent->expectedAttachmentCount += textLayoutManager_.buildParagraph(
-                                                    data.attributedString,
-                                                    props.paragraphAttributes,
-                                                    true,
-                                                    parent->paraBuilder);
+      parent->expectedAttachmentCount += data.layoutManager->buildParagraph(data.attributedString,
```
Comment (Kudo) [inline ReactSkia/components/RSkComponentText.cpp:49]: apologize for late review and having a question here. where is the `data.layoutManager` coming from? maybe some changes are missing from the pr?

Comment (pprabhu-ng) [reply ReactSkia/components/RSkComponentText.cpp:49]: The layoutmanager handle is part of Paragraph State.The data value is read from state which is received in ShadowView. (Refer Line#40)

Comment (Kudo) [reply ReactSkia/components/RSkComponentText.cpp:49]: got it. it's from react-native. thanks for your answer.

**Review by Kudo — APPROVED** [review]
🚀

**Pull request merged**